### PR TITLE
Enhance "go to" feature

### PR DESCRIPTION
**Pull request by PysarenkoDS-BWT**
When we reselect admin level 1 in `Go To` feature we need only values from new admin level 1, but at the moment we added new admin level 1 to array and it's works wrong

Fix it by adding new props and now when we reselect admin level 1, we rewrite array using only selected data

now it works like that
https://user-images.githubusercontent.com/90055175/178262031-70bceffe-5113-4de8-8d09-7f2873436767.mov

### CODE REVIEW

**Comment by ericboucher** [comment]
@PysarenkoDS-BWT so now it works differently for "Go To" and "Alerts", correct?

**Comment by PysarenkoDS-BWT** [comment]
> 

Yep, it's works differently for "Go To" and "Alerts"

**Comment by wadhwamatic** [comment]
Much improved, thanks @PysarenkoDS-BWT! 

I noticed a small issue. If I select an admin area and then use search to enable another second admin area, we seem to have both selected at once. 

Steps to reproduce on http://prism-520.surge.sh/:
1. Select the first admin area in the dropdown (Bantay Chhmar)
2. Use the search field and type in 'phnom penh'. Choose any of the admin areas within that filtered list
3. The resulting map zooms out to show both admin areas at once

This happens regardless of admin level as far as I can tell.

File: src/components/MapView/Layers/BoundaryDropdown.tsx [hunk 1]
```diff
@@ -75,6 +75,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     alignSelf: 'end',
     marginBottom: '0.4em',
   },
+  list: {
+    '& .Mui-selected': { backgroundColor: '#ADD8E6' },
```
[the remark this answers is not in the document]
Comment (ericboucher) [reply src/components/MapView/Layers/BoundaryDropdown.tsx:79]: Give it a try but don't spend too much time if it doesn't work

**Comment by wadhwamatic** [comment]
I retested after the latest commit. Looks good in terms of functionality. Thanks!

**Review by ericboucher — APPROVED** [review]

**Pull request merged**